### ISSUE TITLE
[T3 - P2] Elemento neutro de curva elíptica

### ISSUE THREAD

**Issue by Jaluco** [UC-IIC3253/2022#77]
¿Da igual la representacion que usemos para el elemento neutro? ¿Puedo decir que es el string "O"? o ponerle cualquier nombre? me confunde un poco eso

**Comment by marceloarenassaavedra** [comment]
Sí, puedes elegir la representación que te parezca más apropiada para el elemento neutro. Lo importante es que la implementación de las clases `ElGamalReceiver` y `ElGamalSender` en el notebook funcione con tu representación.
